Website/src/rendering/src/components/Sessions/SessionsGrid.tsx

```diff
@@ -15,16 +16,27 @@ type SessionsGridProps = ComponentProps & {
 };
 
 const SessionsGrid = (props: SessionsGridProps): JSX.Element => {
+  const { sitecoreContext } = useSitecoreContext<SitecoreContextValue>();
+
+  const isPageEditing = sitecoreContext.pageState === LayoutServicePageState.Edit;
+  const hasSessions = props.fields.data?.item;
```

Comment:
`hasSessions` is an object or `null` or `undefined` with that code. We could use `!!props.fields?.data?.item` for it to be a boolean and consume less memory.